buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -1,11 +1,13 @@
 {
   "build": [
     {
+      "environment": {},
```

Comment:
What are these changes?  Are they necessary?

Reply:
These are attributes auto-generated by @chcosta tool to update buildpipeline definition files.